app/models/user.rb

```diff
@@ -521,6 +523,10 @@ def reindex_user_creations_after_rename
     reindex_user_creations
   end
 
+  def add_renamed_at
+    self.renamed_at = Time.now.utc
```

Comment:
With #4169 merged, you can [use `Time.current` here](https://rails.rubystyle.guide/#time-now).